feature-extensions/multiaccount/core.py

```diff
@@ -241,3 +254,40 @@ def preparedestarn(region, account, stackprefix, stage):
                                                                 account,
                                                                 stackprefix,
                                                                 stage, region)
+
+
+def updatePrimaryRole(roleArn, stackprefix,get_configjson):
+    platformRolePrimary = get_configjson['data']['config']['AWS']['PLATFORMSERVICES_ROLEID']
+    permission_policy = {
+        "Version": "2012-10-17",
+        "Statement": [
+            {
+                "Sid": "",
+                "Effect": "Allow",
+                "Action": "sts:AssumeRole",
+                "Resource": roleArn
+            }
+        ]
+    }
+    # Get the default profile
+    session = boto3.Session(profile_name='default')
+    # Create IAM client
+    iamClient = session.client('iam')
+    iam = iamClient.resource('iam')
+    policy = iam.Policy(platformRolePrimary)
+    version = policy.default_version
+    policyJson = version.document
+    # Check if Policy exists
+    if policyJson:
+        policyJson['Statement'].append(permission_policy)
+        iamClient.delete_policy(
+            PolicyArn='arn:aws:iam::%s:policy/%s_NonPrimaryAssumePolicy', %(primary_account,stackprefix)
+        ) 
+    else:
+        policyJson = permission_policy
+
+    iamclient.put_role_policy(
```

Comment:
@Johnbush Remove this extra added inline policy in before invoking stack destroy script (destroy.sh). Otherwise we will get error from terraform like, "DeleteConflict: Cannot delete entity, must delete policies first"

Reply:
Done and raised a PR https://github.com/tmobile/jazz/pull/871